6.4.remember_remember.py

```diff
@@ -0,0 +1,30 @@
+# Author: Omer Gertler
+
+from PIL import Image
+
+DIR = r"C:\Users\omer\Desktop\Notebooks-master\Notebooks-master\week06\resources\code.png"
+
+
+def decipher_msg_from_pic(image_dir):
+    """
+    find all black pixels (numeric value == 1) in the image.
+    searching column by column left to right, each column top down.
+    for every black pixel, it's line-number translated to ascii value.
+    the result is a string message of those decipher line numbers.
+    :param image_dir: the directory where the image stores
+    :return: string of decipher line numbers of the black pixels
+    """
+    black_pixels = []
+    with Image.open(image_dir) as img:
+        width, height = img.size
+        for x in range(width):
+            for y in range(height):
```

Comment:
most of times 'x' and 'y' are not good variable names, although I understand why you chose to use them in this solution